.github/workflows/ci.generate.ts

```diff
@@ -214,6 +213,12 @@ const ci = {
       name: "${{ matrix.job }} ${{ matrix.profile }} ${{ matrix.os }}",
       "runs-on": "${{ matrix.runner || matrix.os }}",
       "timeout-minutes": 120,
+      defaults: {
+        run: {
+          // GH actions doesn't use `set -e` by default unless you specify bash
+          shell: "bash",
```

Comment:
It seems like a terrible decision/oversight that they didn't make this the default https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#exit-codes-and-error-action-preference